flowgate-common/src/main/java/com/vmware/flowgate/common/model/ValueUnit.java

```diff
@@ -62,12 +62,22 @@ public void setKey(String key) {
       this.key = key;
    }
 
+   /**
+    * According to the Metric Prefix in the International System of Units,
+    * M => 1000000; m => 0.001; k => 1000
+    * Link: https://en.wikipedia.org/wiki/International_System_of_Units
+    *
+    */
    public enum MetricUnit {
-       MV("VOLTAGE",0.001), V("VOLTAGE",1), KV("VOLTAGE",1000), VOLTS("VOLTAGE",1),
-       MW("POWER",0.001), W("POWER",1), KW("POWER",1000), KWH("ENERGY",1000),
-       MA("CURRRNT",0.001), A("CURRENT",1), KA("CURRENT",1000), AMPS("CURRENT",1),
-       C("TEMPRETURE",1), F("TEMPRETURE",1), PERCENT("PERCENT",1), BTUPerHr("POWER",0.293071),
-       KB("STORAGE",1), Mhz("FREQUENCY", 1), KBps("RATE", 1);
+       //These units include MV,MW,MA,KW,KA,KWH,KB,KBps,PERCENT will be deprecated in Flowgate-1.3
+       MV("VOLTAGE",0.001), KV("VOLTAGE",1000),MW("POWER",0.001), KW("POWER",1000),
+       KWH("ENERGY",1000), MA("CURRRNT",0.001), KA("CURRENT",1000), PERCENT("PERCENT",1),
+       KB("STORAGE",1), KBps("RATE", 1),
+
+       A("CURRENT",1), V("VOLTAGE",1), W("POWER",1),C("TEMPRETURE",1), VOLTS("VOLTAGE",1),
+       F("TEMPRETURE",1), BTUPerHr("POWER",0.293071), Mhz("FREQUENCY", 1), mV("VOLTAGE",0.001),
+       kV("VOLTAGE",1000), mW("POWER",0.001), kA("CURRENT",1000), percent("PERCENT",1),
```

Comment:
mW MA , double check if we don't use it anywhere, then just remove it .

Reply:
done